.github/dependabot.yaml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "area/dependencies"
+
+  # We maintain updates for most dependencies. This disables updates other than
```

Comment:
How does this disable non-security updates. Seems like a weird API if the default for "no-config" includes non-security updates and the default(?) if there is a config is to include security configs only. I'm just curious.

Reply:
The trick is in setting `open-pull-requests-limit` to 0. The default is a higher number, which allows updates for dependencies to be proposed in new PRs. Setting it to 0 effectively disallows any PRs to be created.

The exception to that limit is for security updates though. So if a security issue needs to be addressed, dependabot will just go ahead with a PR despite whatever the limit number has been set to.